server/.gitignore

```diff
@@ -40,6 +40,7 @@ app_data/
 
 # report builder server credentials
 report_builder/config.yaml
+/**/report_to_show.html
```

Comment:
This command is building reports to the same root dir location, and overwriting when a new report is built. 

I added this so they weren't committed. 

I've put the custom report location saving in a carryover issue [here](https://github.com/msupply-foundation/open-msupply/issues/6404) as it didn't seem critical path to me.